### PR TITLE
chore: Remove deleted Rector rule

### PR DESCRIPTION
**Pull request by brunogaspar**
This removes a freshly deleted Rector rule.

### CODE REVIEW

**Review by owenvoke — APPROVED** [review]

**Comment by owenvoke** [comment]
Looks like the failing tests are unrelated, we need to move to using `doTestFile` instead of `doTestFileInfo` 👍🏻

**Comment by brunogaspar** [comment]
@owenvoke Thank you for the review. Had to also require a new dev dependency, it was probably removed from another package dependency 🤷‍♂️

File: tests/Rector/DisallowedAttributesRector/DisallowedAttributesRectorTest.php [hunk 1]
```diff
@@ -3,19 +3,19 @@
 use Symplify\SmartFileSystem\SmartFileInfo;
 
 it('can remove disallowed attributes', function () {
-    $this->doTestFileInfo(
+    $this->doTestFile(
         new SmartFileInfo(__DIR__ . '/Fixture/class_with_disallowed_attribute.php.inc')
     );
 });
```
Comment (owenvoke) [inline tests/Rector/DisallowedAttributesRector/DisallowedAttributesRectorTest.php:9]: Should be able to replace these with just the string. i.e.

```suggestion
it('can remove disallowed attributes', function () {
    $this->doTestFileInfo(
    $this->doTestFile(
        __DIR__ . '/Fixture/class_with_disallowed_attribute.php.inc'
    );
});
```

Then we don't need the `symplify/smart-file-system` package dependency 👍🏻

Comment (brunogaspar) [reply tests/Rector/DisallowedAttributesRector/DisallowedAttributesRectorTest.php:9]: Thanks, will give it a try :)

**Comment by owenvoke** [comment]
Perfect 👍🏻

**Pull request merged**